### PR TITLE
MQE-1529: Add release to CHANGELOG.MD

### PR DESCRIPTION
**Pull request by KevinBKozan**
### Description
- Added items to changelog
- Version bump in composer

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
 
+2.4.0
+-----
+### Enhancements
+* Maintainability
+    * Added new `mftf static-checks` command to run existing static checks against the attached test codebase
```
Comment (jilu1) [inline CHANGELOG.md]: Why called it "existing" static checks? Isn't it the "new" static checks? We do have some existing static checks and they are different from this one. Does it make sense?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+        * See DevDocs for details
+    * Added new `mftf generate:docs` command that generates documentation about attached test codebase
+        * See DevDocs for details
+* Traceability
+    * Allure reports for tests now contain collapsible sections for `actionGroup`s used in execution.
+
+### Fixes
+* Fixed an issue where `magentoCli` would treat `argument="0"` as a null value.
+* Fixed an issue where `amOnPage` and `waitForPwaElementVisible` would not utilize the `timeout` attribute correctly.
```
Comment (jilu1) [inline CHANGELOG.md]: Not sure if you need to be explicit, the `amOnPage` is also PWA specific.

File: CHANGELOG.md (hunk 1 above)
Comment (KevinBKozan) [reply CHANGELOG.md]: Good point, I'll change the wording here

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,30 @@
...
+* Fixed an issue where invalid XML characters would cause Allure to throw an exception without a resulting report.
+* Fixed `codeception.dist.yml` configuration for keeping previous test run results.
+* PHP Notices are no longer thrown then XML is missing non-necessary attributes.
```
Comment (jilu1) [inline CHANGELOG.md]: typo? "then" -> "when"

File: CHANGELOG.md (hunk 2 above)
Comment (KevinBKozan) [reply CHANGELOG.md]: I didn't work on the ticket so I was summarizing best as I could understand. How would you word this line instead?

File: CHANGELOG.md (hunk 3 above)
Comment (KevinBKozan) [reply CHANGELOG.md]: Good catch 👍

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,30 @@
...
+* Removed unusable `fillSecretField` action from schema.
```
Comment (jilu1) [inline CHANGELOG.md]: Action group context (MQE-1472) is not mentioned. Is this on purpose?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,30 @@
...
+
+### GitHub Issues/Pull requests:
+* [#338](https://github.com/magento/magento2-functional-testing-framework/pull/338) -- Return exit codes of process started by 'run:test', 'run:group' or 'run:failed' command
+* [#333](https://github.com/magento/magento2-functional-testing-framework/pull/333) -- Added Nginx specific settings to getting started doc
+* [#318](https://github.com/magento/magento2-functional-testing-framework/pull/318) -- Reduce cyclomatic complexity in Problem Methods
+
```
Comment (jilu1) [inline CHANGELOG.md]: What about MQE-1479? PHP 7.3 support.

Comment (jilu1) [inline CHANGELOG.md]: And MQE-1521 for executeInSelenium() generation fix?

Comment (KevinBKozan) [reply CHANGELOG.md]: Good catch as well 👍

Comment (KevinBKozan) [reply CHANGELOG.md]: Will do 👍

File: CHANGELOG.md (hunk 4 above)
Comment (KevinBKozan) [reply CHANGELOG.md]: MQE-1472 was addressed in line 13

File: CHANGELOG.md (hunk 2 above)
Comment (jilu1) [reply CHANGELOG.md]: Something like:
Fixed an issue where `amOnPage` and `waitForPwaElementVisible` would not utilize the `timeout` attribute correctly when MagentoPwaWebDriver is enabled.

Comment (KevinBKozan) [reply CHANGELOG.md]: Sounds good 👍

File: CHANGELOG.md (hunk 4 above)
Comment (jilu1) [reply CHANGELOG.md]: Ah, I see

**Pull request merged**